sdk/communication/communication-sms/test/smsClient.mocked.spec.ts

```diff
@@ -45,73 +77,153 @@ describe("[mocked] SmsClient", async () => {
     smsClient = new SmsClient(baseUri, new AzureKeyCredential("banana"), {
       httpClient: mockHttpClient
     });
+
     const spy = sinon.spy(mockHttpClient, "sendRequest");
-    const sendRequest: SendRequest = {
+    sinon.useFakeTimers();
+    const sendRequest: SmsSendRequest = {
       from: "+18768984505651",
       to: ["+18768985487"],
       message: "message"
     };
-    const { _response } = await smsClient.send(sendRequest);
 
+    const _responses = await smsClient.send(sendRequest);
+    const _response = _responses[0];
     sinon.assert.calledOnce(spy);
-    assert.equal(_response.status, 200);
-    assert.equal(_response.parsedBody.messageId, "id");
-
+    assert.equal(_response.httpStatusCode, 202);
+    assert.equal(_response.messageId, "id");
     const request = spy.getCall(0).args[0];
-
     assert.equal(request.url, `${baseUri}/sms?api-version=${API_VERSION}`);
     assert.equal(request.method, "POST");
-    assert.deepEqual(JSON.parse(request.body), { ...sendRequest, sendSmsOptions: {} });
+
+    const expectedRequestBody = {
+      from: sendRequest.from,
+      smsRecipients: [
+        {
+          to: "+18768985487",
+          repeatabilityFirstSent: new Date().toUTCString(),
+          repeatabilityRequestId: mockedGuid
+        }
+      ],
+      message: sendRequest.message,
+      smsSendOptions: {
+        enableDeliveryReport: false
+      }
+    };
+
+    assert.deepEqual(JSON.parse(request.body), expectedRequestBody);
   });
 
   it("sends SMS when connection string is provided", async () => {
     const connectionString = `endpoint=${baseUri};accesskey=banana`;
     smsClient = new SmsClient(connectionString, { httpClient: mockHttpClient });
+
     const spy = sinon.spy(mockHttpClient, "sendRequest");
-    const sendRequest: SendRequest = {
+    sinon.useFakeTimers();
+    const sendRequest: SmsSendRequest = {
       from: "+18768984505651",
```

Comment:
We should change these numbers to use the 555 scheme, like +15550100 and +15550101
https://en.wikipedia.org/wiki/555_(telephone_number)